config-linux.md

```diff
@@ -261,9 +261,12 @@ For more information, see the kernel cgroups documentation about [memory][cgroup
 
 Values for memory specify the limit in bytes, or `-1` for unlimited memory.
 
-* **`limit`** *(int64, OPTIONAL)* - sets limit of memory usage
-* **`reservation`** *(int64, OPTIONAL)* - sets soft limit of memory usage
+* **`limit`** *(int64, OPTIONAL)* - sets limit of memory usage.  It maps to `memory.max` on cgroup v2
```

Comment:
Let's show the v1 mappings explicitly as well

Reply:
(Maybe in tables)

Reply:
should I copy the table from https://github.com/containers/crun/blob/master/crun.1.md#cgroup-v2 (and leave only the ones that have a conversion to a different range)?

Reply:
I've specified the file name on cgroup v1 as well.